api/base/parsers.py

```diff
@@ -25,14 +50,31 @@ def parse(self, stream, media_type=None, parser_context=None):
         data = result.get('data', {})
 
         if data:
-            if 'attributes' not in data:
-                raise JSONAPIException(source={'pointer': '/data/attributes'}, detail=NO_ATTRIBUTES_ERROR)
+            relationships = data.get('relationships')
+
+            path = stream.path
+            related_resources = ['contributors', 'node_links']
```

Comment:
Can you think of any way to avoid hard coded values like this? I'm pretty unfamiliar with DRF parsers, but am wondering if we have access to the serializer in this context?